README.md

```diff
@@ -18,11 +42,13 @@ This tool has been developed in conjuction with the University of Canterbury Com
         user: <github_username>
 ```
 
-* Next save that config file as "swarm_updater_config" in your swarm, this can be done <br />
-by typing ```docker config create swarm_updater_config <path-to-file>```
+2. Next save that config file as "swarm_updater_config" and set it as a config file in your swarm, this can be done <br />
+by typing ```docker config create swarm_updater_config <path-to-file>``` or can be setup as part of step 3 in your deployment compose file.
```

Comment:
I can't get this suggestion to format properly, but have all the text on one line, then the code following in a code block (three backtick delimiters). This avoids using HTML within the Markdown.

```
2. Next save that config file as "swarm_updater_config" and set it as a config file in your swarm, this can be done by typing the following command (or can be setup as part of step 3 in your deployment compose file).
   docker config create swarm_updater_config <path-to-file>
```